student-store-ui/src/components/ProductCard/ProductCard.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import "./ProductCard.css";
+import { BrowserRouter, Link, Route, Routes } from "react-router-dom";
+
+export default function ProductCard({
+  product,
+  productId,
+  quantity,
+  handleAddItemToCart,
+  handleRemoveItemFromCart,
+  showDescription = false,
+}) {
+  const classNameAppend = showDescription ? "view-" : "";
+  const addItem = (event) => handleAddItemToCart(productId)
+  const removeItem = (event) => handleRemoveItemFromCart(productId)
+  return (
+    <div className={`${classNameAppend}product-card`}>
```

Comment:
Given large use of `div`, could opt for more semantic tag names such as `section`